<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransitionFunction</class>
 <widget class="QDialog" name="TransitionFunction">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>166</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextEdit" name="te_tf">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>551</width>
     <height>121</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustToContents</enum>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stack_tf">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>601</width>
     <height>21</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tf0">
    <widget class="QTextBrowser" name="te_tf0_name">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>591</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Courier New'; font-size:9pt; font-weight:600; color:#cc7832;&quot;&gt;def &lt;/span&gt;&lt;span style=&quot; font-family:'Courier New'; font-size:9pt; font-weight:600; color:#1e2124;&quot;&gt;transition_function(&lt;/span&gt;&lt;span style=&quot; font-family:'Courier New'; font-size:9pt; font-weight:600; color:#005500;&quot;&gt;state&lt;/span&gt;&lt;span style=&quot; font-family:'Courier New'; font-size:9pt; font-weight:600; color:#1e2124;&quot;&gt;, &lt;/span&gt;&lt;span style=&quot; font-family:'Courier New'; font-size:9pt; font-weight:600; color:#005500;&quot;&gt;factor&lt;/span&gt;&lt;span style=&quot; font-family:'Courier New'; font-size:9pt; font-weight:600; color:#1e2124;&quot;&gt;):   &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tf1">
    <widget class="QTextBrowser" name="te_tf1_name">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>591</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Courier New'; font-size:9pt; font-weight:600; color:#cc7832;&quot;&gt;cdef double &lt;/span&gt;&lt;span style=&quot; font-family:'Courier New'; font-size:9pt; font-weight:600; color:#1e2124;&quot;&gt;transition_function(float &lt;/span&gt;&lt;span style=&quot; font-family:'Courier New'; font-size:9pt; font-weight:600; color:#005500;&quot;&gt;state&lt;/span&gt;&lt;span style=&quot; font-family:'Courier New'; font-size:9pt; font-weight:600; color:#1e2124;&quot;&gt;, float &lt;/span&gt;&lt;span style=&quot; font-family:'Courier New'; font-size:9pt; font-weight:600; color:#005500;&quot;&gt;factor&lt;/span&gt;&lt;span style=&quot; font-family:'Courier New'; font-size:9pt; font-weight:600; color:#1e2124;&quot;&gt;):   &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tf2">
    <widget class="QTextBrowser" name="te_tf2_name">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>591</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Courier New'; font-size:9pt; font-weight:600; color:#cc7832;&quot;&gt;cdef double &lt;/span&gt;&lt;span style=&quot; font-family:'Courier New'; font-size:9pt; font-weight:600; color:#1e2124;&quot;&gt;transition_function(tuple &lt;/span&gt;&lt;span style=&quot; font-family:'Courier New'; font-size:9pt; font-weight:600; color:#005500;&quot;&gt;state&lt;/span&gt;&lt;span style=&quot; font-family:'Courier New'; font-size:9pt; font-weight:600; color:#1e2124;&quot;&gt;, float &lt;/span&gt;&lt;span style=&quot; font-family:'Courier New'; font-size:9pt; font-weight:600; color:#005500;&quot;&gt;factor&lt;/span&gt;&lt;span style=&quot; font-family:'Courier New'; font-size:9pt; font-weight:600; color:#1e2124;&quot;&gt;):   &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tf3">
    <widget class="QTextBrowser" name="te_tf3_name">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>521</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Courier New'; font-size:9pt; font-weight:600; color:#cc7832;&quot;&gt;cdef double &lt;/span&gt;&lt;span style=&quot; font-family:'Courier New'; font-size:9pt; font-weight:600; color:#1e2124;&quot;&gt;transition_function(float &lt;/span&gt;&lt;span style=&quot; font-family:'Courier New'; font-size:9pt; font-weight:600; color:#005500;&quot;&gt;state&lt;/span&gt;&lt;span style=&quot; font-family:'Courier New'; font-size:9pt; font-weight:600; color:#1e2124;&quot;&gt;, tuple &lt;/span&gt;&lt;span style=&quot; font-family:'Courier New'; font-size:9pt; font-weight:600; color:#005500;&quot;&gt;factor&lt;/span&gt;&lt;span style=&quot; font-family:'Courier New'; font-size:9pt; font-weight:600; color:#1e2124;&quot;&gt;):   &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tf4">
    <widget class="QTextBrowser" name="te_tf4_name">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>591</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Courier New'; font-size:9pt; font-weight:600; color:#cc7832;&quot;&gt;cdef double &lt;/span&gt;&lt;span style=&quot; font-family:'Courier New'; font-size:9pt; font-weight:600; color:#1e2124;&quot;&gt;transition_function(tuple &lt;/span&gt;&lt;span style=&quot; font-family:'Courier New'; font-size:9pt; font-weight:600; color:#005500;&quot;&gt;state&lt;/span&gt;&lt;span style=&quot; font-family:'Courier New'; font-size:9pt; font-weight:600; color:#1e2124;&quot;&gt;, tuple &lt;/span&gt;&lt;span style=&quot; font-family:'Courier New'; font-size:9pt; font-weight:600; color:#005500;&quot;&gt;factor&lt;/span&gt;&lt;span style=&quot; font-family:'Courier New'; font-size:9pt; font-weight:600; color:#1e2124;&quot;&gt;):   &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="bu_right">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>6</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="bu_left">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>6</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
